scout/adapter/mongo/query.py

```diff
@@ -469,8 +469,10 @@ def coordinate_filter(self, query, mongo_query):
             mongo_query(dict): returned object contains coordinate filters
 
         """
-        mongo_query["position"] = {"$lte": int(query["end"])}
-        mongo_query["end"] = {"$gte": int(query["start"])}
+        start_pos = int(query["start"]) if query["start"] >= 1 else 1
```

Comment:
Perhaps fixing the query itself is an overkill, since you shouldn't be able to enter coords <1 now. What do you think?

Reply:
In general I'd say a backend should never rely on getting sensible stuff from the frontend, so all good with a check/fix of faulty input. It's not like we have all endpoints fully locked down for the logged in users.